<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generate Simulation Plots</string>
  </property>
   
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>580</width>
     <height>240</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
     
     <item row="0" column="0">
       <widget class="QLabel" name="label_2">
         <property name="text">
           <string>Input shape-file</string>
         </property>
       </widget>
     </item>
     <item row="0" column="1">
       <widget class="QLineEdit" name="tbInputShpFN"/>
     </item>
     <item row="0" column="2">
       <widget class="QPushButton" name="btnBrowseInputShp">
         <property name="text">
           <string>Browse</string>
         </property>
       </widget>
     </item>
     
     <item row="1" column="0">
       <widget class="QLabel" name="label_6">
         <property name="text">
           <string>Polygon ID</string>
         </property>
       </widget>
     </item>
     <item row="1" column="1">
       <widget class="QComboBox" name="cmbFields"/>
     </item>
     
     <item row="2" column="0">
       <widget class="QLabel" name="label">
         <property name="text">
           <string>Output shape-file</string>
         </property>
       </widget>
     </item>
     <item row="2" column="1">
       <widget class="QLineEdit" name="tbOutputShpFN"/>
     </item>
     <item row="2" column="2">
       <widget class="QPushButton" name="btnBrowseOutputShp">
         <property name="text">
           <string>Browse</string>
         </property>
       </widget>
     </item>
     
     <item row="3" column="0">
       <widget class="QLabel" name="label_3">
         <property name="text">
           <string>Shape</string>
         </property>
       </widget>
     </item>
     <item row="3" column="1">
     <widget class="QComboBox" name="cmbShape">
       <item>
         <property name="text">
           <string>best</string>
         </property>
       </item>
       <item>
       <property name="text">
        <string>square</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>circle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>rectangle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ellipse</string>
       </property>
      </item>
       <property name="currentIndex">
         <number>1</number>
       </property>
     </widget>
    </item>
     
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Position</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="cmbPosition">
      <item>
       <property name="text">
        <string>centroid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bounding box</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mean coordinates</string>
       </property>
      </item>
     </widget>
    </item>
     
     <item row="5" column="0">
       <widget class="QLabel" name="label_5">
         <property name="text">
           <string>Placement</string>
         </property>
       </widget>
     </item>
     <item row="5" column="1">
       <widget class="QComboBox" name="cmbPlacement">
         <item>
           <property name="text">
             <string>optimized</string>
           </property>
         </item>
         <item>
           <property name="text">
             <string>rotated</string>
           </property>
         </item>
         <item>
           <property name="text">
             <string>translated</string>
           </property>
         </item>
         <item>
           <property name="text">
             <string>fixed</string>
           </property>
         </item>
       </widget>
     </item>
      </layout>
    </widget>
   
    <widget class="QDialogButtonBox" name="btnBox">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>260</y>
          <width>580</width>
          <height>30</height>
        </rect>
      </property>
      <property name="orientation">
        <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
        <bool>true</bool>
      </property>
    </widget>
   
 </widget>
  
 <resources/>
 <connections>
  <connection>
   <sender>btnBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
